File: IT/2/Input-Output/Esercizi/C++/esercizio11.cpp
/*
file esercizio11.cpp
compilare con: g++ -std=c++11 esercizio11.cpp
*/

#include <iostream>
#include <fstream>

using std::cin;
using std::cout;
using std::endl;
using std::string;
using std::ofstream;

int main()
{
	cout << "Scrivi una parola o una frase: ";

	string s;
	getline(cin, s);

	string filename = "file.txt";
	ofstream doc(filename);

	doc << s << endl;

	cout << "File \"" << filename << "\" salvato su disco." << endl;
}

File: IT/2/Input-Output/Esercizi/Java/Esercizio11.java
/* file Esercizio11.java */

import java.io.File;
import java.io.FileWriter;
import java.io.BufferedWriter;
import java.io.IOException;

public class Esercizio11 {
	public static void main(String [] args) {
		String s = System.console().readLine("Scrivi una parola o una frase: ");

		try {
			File doc = new File("file.txt");
			FileWriter writer = new FileWriter(doc);
			BufferedWriter buffer = new BufferedWriter(writer);

			buffer.write(s);
			buffer.write(System.lineSeparator());
			buffer.close();

			System.out.println("File \"" + doc.getName() + "\" salvato su disco.");
		}
		catch (IOException e) {
			System.out.println(e);
		}
	}
}

